consumer.go

```diff
@@ -2,11 +2,14 @@ package cluster
 
 import (
 	"sort"
+	"regexp"
 	"time"
 
 	"gopkg.in/Shopify/sarama.v1"
 )
 
+var validChars *regexp.Regexp = regexp.MustCompile(`\A[A-Za-z0-9._-]*\z`)
```

Comment:
Yes.
> Are you sure consumer group names can start with non alphanumeric characters?

According to [this](https://github.com/apache/kafka/blob/b62f8ea43b6d5307f7274fbe8b7984dd5ee22239/core/src/main/scala/kafka/common/Config.scala#L25-L35), the group ID is permitted to start with a `[._-]`.